mmv1/templates/terraform/examples/dataform_config.tf.tmpl

```diff
@@ -0,0 +1,40 @@
+data "google_project" "project" {
```

Comment:
In our tests, we share a single project (per environment). Because this is a singleton on that project, and all that's being changed is the kms key, I think you will need to instead create a dedicate project per-test run. If you don't the kms key would need to be bootstrapped (otherwise they will accumulate), and then the test will not change anything on each run.